<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    <param name="type" value="udp" />
    <param name="topic" value="scan"/>
    <param name="Savelog" value="false"/>  
    <param name="logPath" value="/tmp/log"/>
    <param name="platform" value="LDS-50C-E" />
    <param name="lidar_port" value="6543" />
    <param name="local_port" value="6668" />
    <param name="lidar_ip" value="192.168.158.98" />
    <param name="group_listener" value="false" />
    <param name="group_ip" value="224.0.0.11" />
    <param name="raw_bytes" value="3"/>
    <param name="frame_id" value="map" />
    <param name="rpm" value="1500"/>
    <param name="min_dist" value="0.1"/>
    <param name="max_dist" value="50.0"/>
    <param name="zero_shift" value="0"/>
    <param name="collect_angle" value="0"/>  
    <param name="from_zero" value="false"/>
    <param name="unit_is_mm" value="true"/>
    <param name="output_360" value="true" />
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="with_smooth" value="true" />
    <param name="with_deshadow" value="true" />
    <param name="with_confidence" value="true"/>
    <param name="with_checksum" value="true"/>
    <param name="alarm_msg" value="false"/>
    <param name="inverted" value="false"/>
    <param name="reversed" value="false"/>
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="-3.1415926"/>
    <param name="max_angle" value="3.1415926"/> 
    <rosparam param="mask1" >[1.57,3.14]</rosparam>   
    <param name="hard_resample" value="false"/>
    <param name="soft_resample" value="false"/>
    <param name="resample_res" value="1"/>
    <param name="error_circle" value="3"/>
    <param name="error_scale" value="0.9"/>
    <param name="direction" value="0"/>                 
  </node>
 </group>
</launch>






